Description: Do not require applications to define UPOWER_ENABLE_DEPRECATED to access suspend/resume functionality. We are not ready to do that yet in Debian.
Author: Martin Pitt <mpitt@debian.org>

Index: upower/libupower-glib/up-client.h
===================================================================
--- upower.orig/libupower-glib/up-client.h	2013-03-19 19:09:40.572836704 +0100
+++ upower/libupower-glib/up-client.h	2013-03-19 19:09:36.840836523 +0100
@@ -23,6 +23,9 @@
 #error "Only <upower.h> can be included directly."
 #endif
 
+/* Debian specific: We are not ready yet to deprecate this by default */
+#define UPOWER_ENABLE_DEPRECATED
+
 #ifndef __UP_CLIENT_H
 #define __UP_CLIENT_H
 
